openstudiocore/src/model/OutputMeter.cpp

```diff
@@ -634,6 +652,10 @@ openstudio::OptionalTimeSeries OutputMeter::getData(const std::string& envPeriod
   return getImpl<detail::OutputMeter_Impl>()->getData(envPeriod, specificInstallLocation);
 }
 
+std::vector<IdfObject> OutputMeter::remove() {
```

Comment:
I get a lot of the following linking errors if I dont have that:
error LNK2001: unresolved external symbol "public: virtual class std::vector<class openstudio::IdfObject,class std::allocator<class openstudio::IdfObject> > __cdecl openstudio::model::OutputMeter::remove(void)"